src/modules/site-v2/base/utils/tools.py

```diff
@@ -122,7 +122,11 @@ def try_submit(EntityClass, user, data, no_cache):
       msg += f' (Line: { ex.line })'
 
     # Return the error message
-    return { 'message': msg }, 400
+    return {
+      'message': msg,
+      'full_msg_link': ex.full_msg_link,
+      'full_msg_body': ex.full_msg_body,
+    }, 400
```

Comment:
`400` sounds like a good candidate for an `enum/class`. I.e.: `HttpStatusCode.BAD_REQUEST`

Reply:
Happy to make the enum if you think it would help...  I'm tempted to say the code numbers are pretty transparent, but I can see how they might be confusing, esp. when they're returned a few function calls down (as opposed to from the view function directly)

Reply:
I see -- maybe I'm looking at this from the perspective of new developer approaching the codebase where it might be useful to know the domain of the 'magic numbers' in code, eg: is the number referring to some internal status code of the application layer?Http? others? in this case, it helps to clarify that we are setting the HttpStatusCode semantically in the use of enums (of course, at the expense of extra verbosity 😬).

Reply:
Makes sense. I'll add a card for it in BC